README.md

```diff
@@ -100,6 +100,12 @@ You can also ignore warnings from certain files by setting `ignored_files`:
 ```ruby
 # Ignoring warnings from Pods
 xcode_summary.ignored_files = '**/Pods/**'
+
+# Ignoring specific warnings
+xcode_summary.ignored_results { |result|
```

Comment:
In my opinion, it should be `ignored_results` than `ignored_warnings` because this block also affects for errors.